torchtitan/experiments/deepseek_v3/generate.py

```diff
@@ -299,8 +336,8 @@ def generate_with_cuda_graph(
         {"role": "user", "content": user_prompt},
     ]
 
-    generate(model, pp_schedule, tokenizer, dist_config, messages)
-    generate_with_cuda_graph(model, tokenizer, dist_config, messages)
+    generate(model, pp_schedule, tokenizer, dist_config, messages, use_timer=True)
+    generate_with_cuda_graph(model, tokenizer, dist_config, messages, use_timer=True)
```

Comment:
nit: maybe we don't need `use_timer` arg? That is, when someone use `@time_generation`, then it will always record time. Without the @, then no timing.

Reply:
yes, this would make things cleaner.  Removed the option and streamlined the decorator code.